lib/descartes/version.rb

```diff
@@ -13,5 +13,5 @@
 ##
 
 class Descartes
-  VERSION ||= '0.8.11'
+  VERSION ||= '0.8.12'
```

Comment:
Please avoid to bump versions. If you really want, for this time please bump to 0.9.0.